nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/test/java/org/nuxeo/ecm/platform/comment/TestCommentsMigrator.java

```diff
@@ -182,21 +196,31 @@ protected void testMigration(Runnable migrator) {
         DocumentModel file2 = session.createDocumentModel("/test-domain", "file2", "File");
         file2 = session.createDocument(file2);
 
+        NuxeoPrincipal principal = session.getPrincipal();
         // create some comments
         for (int i = 0; i < NB_COMMENTS; i++) {
             DocumentModel comment = session.createDocumentModel(null, "comment_" + i, COMMENT_DOC_TYPE);
+            DocumentModel createdComment;
             if (i % 2 == 0) {
-                relationCommentManager.createComment(file1, comment);
+                createdComment = relationCommentManager.createComment(file1, comment);
             } else {
-                relationCommentManager.createComment(file2, comment);
+                createdComment = relationCommentManager.createComment(file2, comment);
             }
+            notificationManager.addSubscription(principal.getName(), "notification" + i, createdComment, FALSE,
```

Comment:
Shouldn't we check that no notification was received ?

Reply:
I didn't find any other way than checking that the fired event contains the `DISABLE_NOTIFICATION_SERVICE` parameter.